### PR TITLE
Add iseq

### PR DESCRIPTION
**Pull request by wasade**
cc @charles-cowart

### CODE REVIEW

**Review by charles-cowart — APPROVED** [review]
Approved. No changes required. Looks good, thanks!

**Comment by wasade** [comment]
Thanks!

File: metapool/prep.py [hunk 1]
```diff
@@ -74,15 +76,33 @@ def parse_illumina_run_id(run_id):
...
-    if matches is None or len(matches.groups()) != 2:
+    # iSeq uses YYYYMMDD and has a trailing -XXXX value, for example
+    # 20220303_FS10001773_6_BRB11606-1914
+    # So the regex is updated to allow 8 numbers for the date, and to handle
+    # the trailing -XXXX piece
+    matches8 = re.search(r'^(\d{8})_([\w-]*)', run_id)
+
+    if matches6 is None and matches8 is None:
+        raise ValueError('Unrecognized run identifier format "%s". The '
+                         'expected format is YYMMDD_machinename_XXXX_FC.' %
```
Comment (ElDeveloper) [inline metapool/prep.py:89]: Looks like the error message here should be updated to also mention the format the iSeq follows.

Comment (wasade) [reply metapool/prep.py:89]: Ah, good spot, thank you

**Pull request merged**